Automattic/jetpack
# Admin Page: do not query WAF endpoint when not necessary

#### Changes proposed in this Pull Request:

If the WAF module isn't active:
- We do not need to access or modify settings since those are not displayed in the UI until you flip the toggle and enable the module.
- the endpoints aren't registered yet so you will only get a 404 when querying the endpoint.


#### Other information:


#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Start on a site running latest stable or `trunk`, connected to WordPress.com, where the WAF module is not active.
* Go to Jetpack > Settings with your console open.
* Notice the 404 response from the `jetpack/v4/waf` endpoint request.
* Check out this branch.
* Refresh the page. Notice no `jetpack/v4/waf` request at all.
* Flip the toggle to enable the module.
* Notice the `jetpack/v4/waf` request being made immediately, and the settings appearing.
